stats/view/worker_commands.go

```diff
@@ -151,7 +155,12 @@ func (cmd *recordReq) handleCommand(w *worker) {
 		}
 		ref := w.getMeasureRef(m.Measure().Name())
 		for v := range ref.views {
-			v.addSample(cmd.tm, m.Value())
+			e := &exemplar.Exemplar{
```

Comment:
Creating a new `exemplar` for every `measurement` is very wrong. Going to revert this change.